.travis.yml

```diff
@@ -8,8 +8,6 @@ branches:
   except:
     - demo
 node_js:
-    - 0.10
-    - 0.12
     - 4
     - 5
```

Comment:
Could we make a dedicated PR for this stuff ? We are starting to mix to concerns here :)